docs/core/whats-new/dotnet-9/sdk.md

```diff
@@ -126,3 +126,62 @@ The following two BuildCheck rules are run:
 When a problem is detected, a diagnostic is produced in the build output for the project that contains the issue.
 
 For more information, see the [design documentation](https://github.com/dotnet/msbuild/blob/main/documentation/specs/BuildCheck/BuildCheck.md).
+
+## Analyzer mismatch
+
+Many users install the .NET SDK and Visual Studio at different cadences. While this flexibility is desirable, it can lead to problems for tooling that needs to interop between the two environments. One example of this kind of tooling is Roslyn Analyzers. Analyzer authors have to code for specific versions of Roslyn, but which versions are available and which is used by a given build is sometimes unclear.
+
+This kind of version mismatch between the .NET SDK and MSBuild is referred to as a *torn SDK*. When you're in this torn state, you might see errors like this:
```

Comment:
We probably shouldn't use "torn" twice in the same sentence. This is a bit like using the term for the definition. We use "torn" all the time, but I consider it jargon.

```suggestion
This kind of version mismatch between the .NET SDK and MSBuild is referred to as a *torn SDK*. When you're in this state, you might see errors like this:
```